### PR TITLE
ENT-5195 Fixed tailing of failed feeder log

### PR DESCRIPTION
**Pull request by craigcomstock**
Changelog: none
Ticket: ENT-5195

### CODE REVIEW

**Comment by craigcomstock** [comment]
Should be cherry-picked to 3.12.x right away and 3.15.x after 3.15.0 is released.

File: templates/federated_reporting/config.sh.mustache [hunk 1]
```diff
@@ -36,6 +36,8 @@ CFE_FR_TABLES="${CFE_FR_TABLES:-$DEFAULT_TABLES}"
 # dump phase
 CFE_FR_COMPRESSOR="{{compressor}}"
 CFE_FR_COMPRESSOR="${CFE_FR_COMPRESSOR:-gzip}"
+CFE_FR_DECOMPRESSOR="{{decompressor}}"
+CFE_FR_DECOMPRESSOR="${CFE_FR_DECOMPRESSOR:-zcat}"
```
Comment (vpodzime) [inline templates/federated_reporting/config.sh.mustache]: I'd prefer a `CFE_FR_DECOMPRESS_ARG` and use the `CFE_FR_COMPRESSOR CFE_FR_DECOMPRESS_ARG` to decompress things.

Comment (craigcomstock) [reply templates/federated_reporting/config.sh.mustache]: yeah, I didn't love this naming. :+1: will make a fix for it.

**Comment by olehermanse** [comment]
@vpodzime @craigcomstock Should this still be cherry-picked?

**Comment by craigcomstock** [comment]
It has been cherry picked properly.
3.15.x: https://github.com/cfengine/masterfiles/pull/1627
3.12.x: https://github.com/cfengine/masterfiles/pull/1635

**Pull request merged**